Description: add type to IMPORT keyword in ifrename's udev rules file
Author: Guus Sliepen <guus@debian.org>
Bug-Debian: http://bugs.debian.org/650606

--- wireless-tools-30~pre9.orig/19-udev-ifrename.rules
+++ wireless-tools-30~pre9/19-udev-ifrename.rules
@@ -5,9 +5,9 @@
 # when udev runs (i.e. on boot partition).
 
 # Enable this rule to test with udevtest.
-#ENV{UDEV_LOG}=="6", SUBSYSTEM=="net", ACTION=="add", IMPORT="/sbin/ifrename -D -V -u -i %k", NAME:="$env{INTERFACE}"
+#ENV{UDEV_LOG}=="6", SUBSYSTEM=="net", ACTION=="add", IMPORT{program}="/sbin/ifrename -D -V -u -i %k", NAME:="$env{INTERFACE}"
 
 # Main ifrename rule.
 # If interface is found in /etc/iftab, subsequent rename rules are bypassed.
 # If interface is not found in /etc/iftab, subsequent rename rules applies.
-SUBSYSTEM=="net", ACTION=="add", IMPORT="/sbin/ifrename -u -i %k", NAME:="$env{INTERFACE}"
+SUBSYSTEM=="net", ACTION=="add", IMPORT{program}="/sbin/ifrename -u -i %k", NAME:="$env{INTERFACE}"
